### ISSUE TITLE
Interface replaced by error message

### ISSUE THREAD

**Issue by RobertAndersonHunt-PeterMac** [susom/mass-delete#18]
Hi team,

Mass Delete is showing this error at the bottom of both EM pages:

![image](https://github.com/user-attachments/assets/9dc3318c-0120-4419-8a77-0662b0283644)

Transcribed version of the error:

> ERROR: There is an error in the REDCap plugin that you are currently using. Please contact this plugin's developer and send them the error info listed below so that they may fix this issue. If you do not know who the developer is for this plugin, then notify your REDCap administrator about this issue.
> 
> Error message: Uncaught Error: Call to undefined method RecordDashboard::renderSelectedFormsEvents() in modules/mass_delete_v1.1.8/MassDelete.php:321 Stack trace: #0 modules/mass_delete_v1.1.8/MassDelete.php(303): Stanford\MassDelete\MassDelete->getFormEventList() #1 modules/mass_delete_v1.1.8/page_mass_delete.php(87): Stanford\MassDelete\MassDelete->initEventForms() #2 redcap_v14.5.2/ExternalModules/index.php(151): require('m...') #3 {main} thrown
> File: modules/mass_delete_v1.1.8/MassDelete.php
> Line: 321

This appears to be a different error from the other two issues (16 & 17) currently open.

Our REDCap instance is on PHP v7.4.33 and REDCap v14.5.2

**Comment by ahackma2** [comment]
Yes, we're seeing this same error since upgrading from 14.0.37 to LTS 14.5.8
 Our REDCap instance is on REDCap 14.5.8 · PHP 7.4.33 (Linux/Unix OS) · MariaDB 10.3.23

I see release note indicating it was deprecated because "Bulk Delete was added in v14.5" but it actually did not get included until Standard v14.6. 
We are running LTS and just upgraded to the latest 14.5.x release in August which is based off of Standard 14.5.4
